gammapy/modeling/parameter.py

```diff
@@ -332,6 +332,34 @@ def types(self):
         """Parameter types"""
         return [par.type for par in self]
 
+    @property
+    def minima(self):
```

Comment:
Here I'm leaning 60% towards calling this `mins` , I agree "minima" sounds better, but the connection between `Parameter.min` and `Parameters.mins` seems stronger. Any opinion @registerrier, @AtreyeeS?

Reply:
`maxs` sound very odd, no ?

Reply:
Yes, I agree it sounds odd, but the connection to `Parameter.max` is stronger...maybe @registerrier or @AtreyeeS have a preference, I don't insist in any way...

Reply:
Well, I am not fond of `mins` and `maxs` which do not sound very clear to me. But it is likely that `minima` and `maxima` are also ambiguous w.r.t. optimal parameter values (e.g. the the value which maximizes likelihood or minimize TS).
maybe a more explicit `range_min` and `range_max` or simply keep the `min` and `max` on the `Parameters` as well.

Reply:
I think the logic of `mins` and `maxs` is just like `value` and `values`...what about if we just ignore the plural and mirror what we have for `Parameter`? So we introduce:

- `Parameters.min` and `Parameters.max`
- Rename  to `Parameters.value` (there is e.g. `Quantity.value`, which returns an array as well)

Reply:
ok done